src/components/InventoryGroupDetail/GroupTabDetails.js

```diff
@@ -23,83 +24,78 @@ const GroupTabDetailsWrapper = ({
   activeTab,
   hasEdgeImages,
 }) => {
-  const [tab, setTab] = useState(0);
+  const [groupTabKey, setGroupTabKey] = useState(groupTabKeys.systems);
+  const [hybridInventoryTabKey, setHybridInventoryTabKey] = useState(
+    hybridInventoryTabKeys.conventional.key
+  );
 
   const { hasAccess: canViewHosts } = usePermissionsWithContext(
     REQUIRED_PERMISSIONS_TO_READ_GROUP_HOSTS(groupId)
   );
-  const conventionalSystemsContent = useMemo(
+
+  const ConventionalTabTable = useMemo(
```

Comment:
These should be removed and not memoize rendered components. See https://react.dev/reference/react/useMemo#memoizing-individual-jsx-nodes